src/frequenz/sdk/actor/power_distributing/power_distributing.py

```diff
@@ -351,7 +349,7 @@ async def _set_distributed_power(
             set of batteries that failed.
         """
         tasks = {
-            inverter_id: asyncio.create_task(api.set_power(inverter_id, power))
+            inverter_id: asyncio.create_task(api.set_power(inverter_id, round(power)))
```

Comment:
This is the only conversion from float to int currently required to set the power in the microgrid API (this will no longer needed once the microgrid API is updated to accept float type to set the power).
Is it fine to convert to int with round()? Or should it use something else like int(), floor() or ceil()?

Reply:
You're right, the client seems to be the right place to do the conversion. And I've already updated to use floor() instead